indico/modules/search/client/js/components/SearchApp.jsx

```diff
@@ -214,11 +214,12 @@ SearchOptions.defaultProps = {
   sortOptions: undefined,
 };
 
-export default function SearchApp({category, eventId}) {
+export default function SearchApp({category, eventId, isAdmin}) {
   const eventSearch = eventId !== null;
   const [query, setQuery] = useQueryParams();
   const [activeMenuItem, setActiveMenuItem] = useState(undefined);
-  const {q, sort, ...filters} = query;
+  const {q, sort, admin_override_enabled: adminOverrideEnabledStr = false, ...filters} = query;
```

Comment:
shouldn't it be `'false'`? `JSON.parse(false) == false` but that seems to be only because it converts the arg to a string..